docs/gh-actions/gh-update.md

```diff
@@ -2,7 +2,7 @@
 
 This is only available with Loop 3 and Loop development branch.
 
-!!! info "Time Estimate"
+??? info "`Time Estimate` (click to open/close)"
```

Comment:
Since we want to translate `Time Estimate`, it seems to me that backticks are not necessary here.